### PR TITLE
Add webUI tests for removing user from groups

### LINKED ISSUE #131: webUI acceptance tests for removing users from a group
Now that issue #128 is done, also:

- [ ] test that a user can be removed from a group using the webUI (#132)

Are there any other interesting user attributes that can be modified in the webUI?

### PR DESCRIPTION
**Pull request by dpakach**
Fixes #131

Add acceptance tests for removing users from the group using the webUI.

### CODE REVIEW

File: tests/acceptance/features/lib/UsersPage.php [hunk 1]
```diff
@@ -661,10 +662,11 @@ public function changeUserPassword(Session $session, $user, $password) {
 	 * @param Session $session
 	 * @param string $user
 	 * @param string $password
+	 * @param boolean $add Boolean value to specify weather to add or remove user from the group
```
Comment (phil-davis) [inline tests/acceptance/features/lib/UsersPage.php]: s/weather/wether
I luv English ;)

File: tests/acceptance/features/lib/UsersPage.php [hunk 2]
```diff
@@ -673,16 +675,32 @@ public function addUserToGroup(Session $session, $user, $group) {
...
 			);
 		}
-		$groupLabel->focus();
-		$groupLabel->click();
-		$this->waitForAjaxCallsToStartAndFinish($session);
+		if ($groupLabel === null) {
+			throw new ElementNotFoundException(
+				__METHOD__ .
+				" xpath $this->groupInputXpath " .
+				"could not input for group $group"
```
Comment (phil-davis) [inline tests/acceptance/features/lib/UsersPage.php]: s/not input/not find input

File: tests/acceptance/features/webUIManageUsersGroups/editUsers.feature [hunk 3]
```diff
@@ -57,4 +57,28 @@ Feature: edit users
...
+    And group "grp2" has been created
+    And group "grp3" has been created
+    And user "user0" has been added to group "grp1"
+    And user "user0" has been added to group "grp2"
+    And user "user0" has been added to group "grp3"
+    And the administrator has browsed to the users page
+    When the administrator removes user "user0" from group "grp1" using the webUI
+    And the administrator removes user "user0" from group "grp2" using the webUI
+    Then user "user0" should exist
     And user "user0" should not belong to group "grp1"
```
Comment (individual-it) [inline tests/acceptance/features/webUIManageUsersGroups/editUsers.feature]: need to keep that  step also in line 60

**Comment by phil-davis** [comment]
Backport to core `stable10` https://github.com/owncloud/core/pull/34232

**Pull request merged**